### ISSUE TITLE
3D model evaluation: no labels in mask

### ISSUE THREAD

**Issue by SaiHK** [MouseLand/cellpose#260]
Hello,

I have tried to train a 3D model using cell pose, I used 5 stack and extract XY, XZ, YZ planes to train the model, however when evaluating the model I don't get any labels in the output mask (got array full of 0) 
The resulting flow map is weird so I was thinking that may be the flow map are not good enough to generate some labels in the mask. But I don't know whether it's a problem of training parameters (#epochs, learning rate etc),  or whether there is a problem with my script or my dataset, what do you think?

Also I used mask with labels 0,1,2... for the training is it ok? or should i rather use binary mask for the training?

![image](https://user-images.githubusercontent.com/59837513/117416726-5719f580-af22-11eb-8132-4d7050e7c12b.png)

Here is the code i used to train and evaluate:

```
# Training 
channels = [0,0]
new_model = models.CellposeModel(gpu=True, pretrained_model=False, 
                                 diam_mean=15.0, net_avg=False,
                                 device=None, residual_on=True, 
                                 style_on=True, concatenation=False)

new_model.train(train_data=train_data, train_labels=train_labels,
                           train_files=data_files,test_data=None, test_labels=None,
                           test_files=None, channels=channels, normalize=True,
                           pretrained_model=None,save_path='/content/drive/MyDrive/cellpose/train3D/images',
                           save_every=100,learning_rate=0.2, n_epochs=500, batch_size=2)
```
```
#Evaluate
from cellpose import models
QC_model_path= '/content/drive/MyDrive/cellpose/train3D/models/cellpose_residual_on_style_on_concatenation_off_new_model_2021_05_06_14_46_16.817803'

model = models.CellposeModel(gpu=True, pretrained_model=QC_model_path, torch=True,
                             diam_mean=15.0, net_avg=False, device=None, 
                             residual_on=True, style_on=True, concatenation=False)
channels = [[0,0]]
masks, flows, styles = model.eval(test_data, channels=[0, 0], diameter=15,do_3D=True,anisotropy=1)
```

Thanks!
